### PR TITLE
Add squoosh app

### PR DESCRIPTION
**Pull request by ngvtuan**
##### Commits
- [x] Add squoosh app

##### FreedCamp
- [x] [Add squoosh app](https://freedcamp.com/view/2956183/tasks/39603824)

### CODE REVIEW

**Review by br4in3x — DISMISSED** [review]
I'm thinking we should combine all modals into one reducer, that will look like this:
```
{
  sqoosh: true,
  about: false,
  // ...
}
```
Where key is the modal name, and value is whether it's open or not.

The same will be for the modal component. We are starting to have too many separate modals. What do you think?

**Comment by ngvtuan** [comment]
@br4in3x 

The same will be for the modal component. We are starting to have too many separate modals. What do you think? --> Yes we can improve the code follow that way and i will create new ticket to improve for the modals

File: src/main.ts [hunk 1]
```diff
@@ -134,6 +136,19 @@ const createWindow = async () => {
...
+    })
+    // Create server
+    const server = http.createServer(function onRequest(req, res) {
+      serve(req, res, () => {
+        console.log('Create server')
+      })
+    })
+    // Listen
+    server.listen(5100)
+  })
```
Comment (br4in3x) [inline src/main.ts:151]: I think the server is not necessary. You can request `file:///node_modules/squoosh/build/index.html` (or something like this) directly in the modal

File: package.json [hunk 2]
```diff
@@ -66,8 +67,10 @@
     "redux-thunk": "2.3.0",
     "regenerator-runtime": "0.13.7",
     "request": "2.88.2",
+    "serve-static": "1.14.1",
     "sha256-file": "1.0.0",
     "sql.js": "pastelnetwork/sql.js#230351f",
+    "squoosh": "pastelnetwork/squoosh#dev",
```
Comment (br4in3x) [inline package.json:73]: Can we have a commit hash instead of the branch name, similar how it's done for `sql.js` please?

File: src/main.ts (hunk 1 above)
Comment (ngvtuan) [reply src/main.ts:151]: @br4in3x I have tried with this but it doesn't work, when you tried to add like that, I will be shown 404. Refer this https://github.com/GoogleChromeLabs/squoosh/issues/999

File: package.json (hunk 2 above)
Comment (ngvtuan) [reply package.json]: I have updated to the hash

**Review by br4in3x — APPROVED** [review]

**Pull request merged**